### ISSUE TITLE
At rules for `@font-face` are not ignored

### ISSUE THREAD

**Issue by daanvosdewael** [soluml/at-rule-packer#1]
When using this plugin, rules for `@font-face` are also merged, giving undesirable results.

For example, code like this:
```css
@font-face {
    font-family: 'Open Sans';
    font-style: normal;
    font-weight: 400;
    src: url('/fonts/OpenSans-Regular.woff2') format('woff2');
}

@font-face {
    font-family: 'Open Sans';
    font-style: normal;
    font-weight: 700;
    src: url('/fonts/OpenSans-Bold.woff2') format('woff2');
}
```

…results in this:

```css
@font-face {
    src: url('/fonts/OpenSans-Regular.woff2') format('woff2');
    font-weight: 400;
    font-style: normal;
    font-family: 'Open Sans';
    font-family: 'Open Sans';
    font-style: normal;
    font-weight: 700;
    src: url('/fonts/OpenSans-Bold.woff2') format('woff2');
}
```

The `@font-face` declarations need to be ignored from processing.

**Comment by soluml** [comment]
Yep, that's definitely undesirable! Thanks for the test case, I'll try to get an update out shortly.

**Comment by soluml** [comment]
Should be addressed in version `0.3.0`. Thanks!
